tests/MartinGeorgiev/Doctrine/DBAL/Types/PointArrayTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\MartinGeorgiev\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidPointArrayItemForPHPException;
+use MartinGeorgiev\Doctrine\DBAL\Types\PointArray;
+use MartinGeorgiev\ValueObject\Point as PointValueObject;
```

Comment:
I think there is no need for the alias to `PointValueObject`. You can use the class name directly as `Point`.